animation_nodes/nodes/gpencil/gp_material_output.py

```diff
@@ -0,0 +1,48 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode
+
+modeTypeItems = [
+    ("LINE", "Line", "Draw for strokes using a continuous line", "NONE", 0),
+    ("DOTS", "Dots", "Draw for strokes using separated dots", "NONE", 1),
+    ("BOX", "Boxes", "Draw for strokes using separated rectangle boxes", "NONE", 2)
+]
+
+class GPMaterialOutputNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPMaterialOutputNode"
+    bl_label = "GP Material Output"
+
+    modeType: EnumProperty(name = "Mode Type", default = "LINE",
```

Comment:
This should be an input as well. Same discussion we had before.

Reply:
Done.